### PR TITLE
Severity updates

### PR DESCRIPTION
**Pull request by djaglowski**
## Description of Changes
Addresses #180 and #181 

## **Please check that the PR fulfills these requirements**
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [x] Add a changelog entry (for non-trivial bug fixes / features)
- [x] CI passes

### CODE REVIEW

**Comment by djaglowski** [comment]
| Log Files | Logs / Second | CPU Avg (%) | CPU Avg Δ (%) | Memory Avg (MB) | Memory Avg Δ (MB) |
| --- | --- | --- | --- | --- | --- |
|1|1000|1.7931392|-0.034489036|128.32207|+0.25889587|
|1|5000|6.1725698|+0.017214775|136.70232|+0.77371216|
|1|10000|12.362321|-0.41380405|145.59428|+1.5701752|
|1|50000|62.55828|+3.2293663|176.0423|+6.6209564|
|1|100000|121.41499|+1.0719604|233.86314|+1.6651306|
|10|100|2.4655986|-0.12064028|134.87042|+3.6170502|
|10|500|6.741515|-0.3103242|141.90436|+5.037033|
|10|1000|13.758966|-1.1725245|146.14601|+1.9873352|
|10|5000|61.587814|-4.6359024|180.74043|-4.7460938|
|10|10000|125.613495|-0.64341736|228.9534|-3.9307556|

**Comment by djaglowski** [comment]
| Log Files | Logs / Second | CPU Avg (%) | CPU Avg Δ (%) | Memory Avg (MB) | Memory Avg Δ (MB) |
| --- | --- | --- | --- | --- | --- |
|1|1000|1.7241943|-0.10343397|128.5722|+0.5090332|
|1|5000|5.8967876|-0.25856733|134.54958|-1.3790283|
|1|10000|12.189867|-0.58625793|142.28488|-1.7392273|
|1|50000|59.430984|+0.102069855|181.7663|+12.344955|
|1|100000|118.08884|-2.2541962|239.81412|+7.616104|
|10|100|2.4138365|-0.17240238|134.37958|+3.1262054|
|10|500|7.138078|+0.08623886|139.33647|+2.4691467|
|10|1000|13.345162|-1.5863285|149.00108|+4.842407|
|10|5000|62.414192|-3.8095245|188.19478|+2.708252|
|10|10000|128.08159|+1.8246765|236.98384|+4.0996857|

File: entry/entry.go [hunk 1]
```diff
@@ -7,11 +7,12 @@ import (
...
 // Entry is a flexible representation of log data associated with a timestamp.
 type Entry struct {
-	Timestamp time.Time         `json:"timestamp"        yaml:"timestamp"`
-	Severity  Severity          `json:"severity"         yaml:"severity"`
-	Labels    map[string]string `json:"labels,omitempty" yaml:"labels,omitempty"`
-	Resource  map[string]string `json:"resource,omitempty" yaml:"resource,omitempty"`
-	Record    interface{}       `json:"record"           yaml:"record"`
+	Timestamp    time.Time         `json:"timestamp"          yaml:"timestamp"`
+	Severity     Severity          `json:"severity"           yaml:"severity"`
+	SeverityText string            `json:"severity_text"      yaml:"severity_text"`
```
Comment (camdencheek) [inline entry/entry.go:12]: Should this have `omitempty` on it?

**Review by camdencheek — APPROVED** [review]

File: entry/entry.go (hunk 1 above)
Comment (djaglowski) [reply entry/entry.go:12]: Yeah, good call. Will update.

**Comment by djaglowski** [comment]
| Log Files | Logs / Second | CPU Avg (%) | CPU Avg Δ (%) | Memory Avg (MB) | Memory Avg Δ (MB) |
| --- | --- | --- | --- | --- | --- |
|1|1000|1.7758838|-0.05174446|128.55038|+0.48721313|
|1|5000|6.310418|+0.15506315|138.60129|+2.6726837|
|1|10000|13.397107|+0.62098217|145.52087|+1.4967651|
|1|50000|57.7275|-1.6014137|172.90181|+3.4804688|
|1|100000|118.242035|-2.100998|229.99515|-2.2028656|
|10|100|2.4310722|-0.15516663|132.95312|+1.6997528|
|10|500|7.293252|+0.24141264|139.26872|+2.4013977|
|10|1000|13.3968525|-1.5346384|143.57288|-0.5858002|
|10|5000|65.77723|-0.44648743|184.32085|-1.1656799|
|10|10000|124.61991|-1.637001|226.59738|-6.2867737|

**Review by camdencheek — APPROVED** [review]

**Comment by djaglowski** [comment]
| Log Files | Logs / Second | CPU Avg (%) | CPU Avg Δ (%) | Memory Avg (MB) | Memory Avg Δ (MB) |
| --- | --- | --- | --- | --- | --- |
|1|1000|1.7414086|+0.034490347|129.50243|-0.16972351|
|1|5000|5.931152|-0.17240715|134.94626|-2.7253418|
|1|10000|11.897084|-0.6031933|143.5462|-1.9894867|
|1|50000|57.81954|-10.061489|175.01549|-8.462967|
|1|100000|114.300026|-4.3642654|225.79944|-10.890488|
|10|100|2.4310696|+0.051720142|132.42377|+0.16595459|
|10|500|6.7242594|-0.2069192|138.34967|+1.7231903|
|10|1000|13.500685|+0.017626762|148.5625|+1.4193115|
|10|5000|62.08863|-1.0849495|181.33392|+2.7023163|
|10|10000|126.58128|+0.5425949|234.45609|+21.76535|

**Pull request merged**